### DISCUSSION TITLE
[web] Missing documents about setting up Nativewind for expo-web with expo-router

### DISCUSSION THREAD

**Discussion by MikeChongCan** [expo/router#389] [Q&A]
https://next.nativewind.dev/quick-starts/expo-web seems to be outdated.

After setting up CSS support with Metro, it does not seem to work: https://github.com/expo/router/blob/152bc8cb08399dda9c029a32345c7d82706b4504/docs/docs/lab/styles.md

**Comment by cseelus** [comment]
Nativewind docs state that it isn‘t compatible with

- Expo Metro: As it [does not support bundling CSS](https://docs.expo.dev/guides/customizing-metro/#expo-webpack-vs-expo-metro)
- Expo Router: It requires Expo Metro
